src/components/Header/Header.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../themes/oneui/oneui.scss';
+
+.Header {
+    width: 100%;
```

Comment:
Missing `background-color`: it will be transparent now. It would also be good to add a component-level variable for that, so you can have whichever header background color you want.